ci/Jenkinsfile

```
@@ -109,15 +106,11 @@ pipeline {
 
 def launchLocalTestnet(String name) {
   /* We want to mark job as failed, but save the results. */
```

This makes this comment incorrect, since the error will not actually be caught by the job like this.

---

Are we sure we want to ignore errors from failed testnet launches? Seems wrong.

---

The stage will still fail on error. It is `try` / `finally`, not `try` / `catch`

---

Are you sure about that? I don't think that's true. Can we test it?